### ISSUE TITLE
AppState values

### ISSUE THREAD

**Issue by cknitt** [reason-react-native/reason-react-native#725]
In the current bindings, the following values are defined for `AppState.t`:

* active
* background
* inactive

However, the TypeScript bindings have

```ts
export type AppStateStatus = 'active' | 'background' | 'inactive' | 'unknown' | 'extension';
```

(see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a65f7cba4823a1c4bf30f56c55e3dde51be33237/types/react-native/index.d.ts#L7294).

I wonder where the additional two are coming from. The React Native docs (https://facebook.github.io/react-native/docs/appstate#app-states) do not mention them.

**Comment by MoOx** [comment]
I would say it's for iOS widget? https://github.com/facebook/react-native/blob/976f51abd907cb2ff8c5fa8fffa1d0abd7cc7222/React/CoreModules/RCTAppState.mm#L23-L30

**Comment by cknitt** [comment]
Ah yes, thanks!

Will add the two values in a separate PR after #724 is merged.
